01week/bio/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      Bio
+    </title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway"
+    rel="stylesheet">
+  </head>
+  <body>
+    <h1>
+      A bit about Stu.
+    </h1>
+    <img src="stu-pool.jpg" alt="stu" class="pic pic-stu">
+    <p class="bio">
+      I have had an interest in computers and coding since I was younger.
+      My mother worked for HP, so I always had a computer in front of me
+      to experiment with. While I've only dabbled in coding thus far,
+      what little HTML I learned before is quickly coming back to me.
+      I have a passion for learning, be it coding, languages, or how to
+      better myself, I'm always excited to learn! I'm an avid gamer, and
+      double-dog-dad.
+    </p>
+    <img src="dogs-couch.JPG" alt="dogs" class="pic pic-dogs">
+    <p class="bio">
+      These are my dogs, Pi (left) and Churchill (right). Churchill likes
+      to stare at my guests until they pet him, while Pi licks the air
+      when she's excited. <strong>WHY WON'T THIS PICTURE COOPERATE</strong>
```

Comment:
I did! And the weird thing is that the picture doesn't display as sideways when viewed from a mobile device.